platform/cli/templates/mode/src/initToolGroups.js

```diff
@@ -0,0 +1,260 @@
+function initDefaultToolGroup(
+  extensionManager,
+  toolGroupService,
+  commandsManager,
+  toolGroupId
+) {
+  const utilityModule = extensionManager.getModuleEntry(
+    '@ohif/extension-cornerstone.utilityModule.tools'
+  );
+
+  const { toolNames, Enums } = utilityModule.exports;
+
+  const tools = {
```

Comment:
Consider exporting some of these sets of values from the appropriate mode, and inheritting them, and then just re-use them.  The toolNames doesn't need to come from the utility module, so it can just be directly included, then this can all be statically defined, and the new mode creator can choose to use the defaults or modify it as they please.